src/overview/components/ResultList.jsx

```diff
@@ -27,6 +27,17 @@ const timeGapToSpaceGap = makeNonlinearTransform({
 })
 
 const ResultList = ({searchResult}) => {
+
+    //if no result found display relavant message
+    if(searchResult.rows.length === 0){
+        return (
+            <p className={styles.message}>
+            no result found
```

Comment:
Talking about style, please do indent this line too. Also "no results found" sounds slightly wrong, I would either say "No results", or "No pages found" or something like that.

I now realise it would be good to make a special case for when the database is empty. "No results" is not a nice welcome message on a fresh install. Perhaps move the current code to the Overview instead of the ResultList, then test for both the length of searchResult and the existence of a query, and if both are empty show something nice, e.g. "This is your digital memory, watch it grow as you browse the web!"